### PR TITLE
Update Telemetry Link to point to ABS docs

### LINKED ISSUE #1935: Documentation for Telemetry / Analytics needs updated
Currently, the documentation for telemetry is not inclusive of all telemetry information.  You can find the out of date documentation here:
https://github.com/microsoft/botframework-solutions/blob/master/docs/reference/analytics/telemetrylogging.md

The documentation should reference the ABS telemetry information to ensure that the latest telemetry information is communicated, which is located here:
https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-telemetry?view=azure-bot-service-4.0

We should separate the VA specific information from the ABS telemetry details if there is specific content to be provided for VA.

### PR DESCRIPTION
**Pull request by Dewain27**
Close #1935 

### Purpose
Changed to address the fact that Telemetry documentation is owned by ABS now and this should redirect to them.

### Changes
No

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

**Comment by Dewain27** [comment]
@ryanlengel please verify that I linked to the proper location for this... Thx

File: docs/readme.md [hunk 1]
```diff
@@ -99,7 +99,7 @@
 | ---- | ----------- |
 |[Application Insights](/docs/reference/analytics/applicationinsights.md)|Detailed information on how Application Insights is used to collect information and powers our Analytics capabilities.|
 |[Power BI Template](/docs/reference/analytics/powerbi.md)|Detailed information on how the provided PowerBI template provides insights into your Assistant usage.|
-|[Telemetry](/docs/reference/analytics/telemetrylogging.md)|How to configure a telemetry collection for your Assistant.|
```
Comment (lzc850612) [inline docs/readme.md:102]: do we still use telemetrylogging.md anywhere else? if not we could remove it.

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: docs/readme.md (hunk 1 above)
Comment (ryanlengel) [reply docs/readme.md]: It's also referenced in /docs/overview/analytics.md which I'm not sure if we need as well.

**Comment by ryanlengel** [comment]
New documentation location looks correct, I don't think we need the Application Insights doc either. It will get out of date as the SDK updates.

File: docs/readme.md (hunk 1 above)
Comment (ryanisgrig) [reply docs/readme.md:102]: It's also referenced in /docs/overview/analytics.md which should be cleaned up to reflect these changes.

**Comment by ryanisgrig** [comment]
New documentation location looks correct.
I don't think we need the Application Insights doc either, but will defer to @scheyal on that.

**Review by darrenj — APPROVED** [review]

**Comment by lauren-mills** [comment]
Pulling this into #2071

**Pull request closed without merging**